src/mapml/layers/MapMLLayer.js

```diff
@@ -144,6 +145,18 @@ export var MapMLLayer = L.Layer.extend({
                   var c = document.createElement('div');
                   c.classList.add("mapml-popup-content");
                   c.insertAdjacentHTML('afterbegin', properties.innerHTML);
+                  c.insertAdjacentHTML('beforeend', `<a href="" class="zoomLink">Zoom to here</a>`);
```

Comment:
Found a bug - Pressing shift+tab when focusing back to the "Zoom to here" link in the popup -> dismisses the popup. I believe this is related to #507.

Reply:
Can also localize this in a separate issue #821, after this is merged

Reply:
I think the links may be getting cached or something of that nature. Does not function properly when there are two layers. 

For ex. paste the code below to index.html and zoom to the geometrycollection layer. Zooming to the point layer now zooms to the geometrycollection. If you refresh the page, and zoom the point first, it will get 'cached' or so i think.

```html
<mapml-viewer projection="OSMTILE" zoom="10" lon="-75.7" lat="45.4" controls>
      <layer- label="OpenStreetMap" src="https://maps4html.org/web-map-doc/demo/data/osm.mapml" checked></layer->
      <layer- label="Point Geometry" checked>
        <map-meta name="projection" content="OSMTILE"></map-meta>
        <map-feature>
          <map-featurecaption>Point</map-featurecaption>
          <!-- Setting the geometry coordinates to gcrs, as the default is pcrs -->
          <map-geometry cs="gcrs">
            <map-point class="point">
              <map-coordinates>-75.4916809 45.4186964</map-coordinates>
            </map-point>
          </map-geometry>
          <map-properties><h2>This is a Point</h2></map-properties>
        </map-feature>
      </layer->
      <layer- label="Geometry Collection" checked>
    <map-meta name="projection" content="OSMTILE"></map-meta>
    <map-feature>
      <map-featurecaption>Geometry Collection</map-featurecaption>
      <!-- Setting the geometry coordinates to gcrs, as the default is pcrs -->
      <map-geometry cs="gcrs">
        <map-geometrycollection>
          <map-polygon class="polygon">
            <map-coordinates>-75.5859375 45.4656690 -75.6813812 45.4533876 -75.6961441 45.4239978 -75.7249832 45.4083331 -75.7792282 45.3772317 -75.7534790 45.3294614 -75.5831909 45.3815724 -75.6024170 45.4273712 -75.5673981 45.4639834 -75.5859375 45.4656690</map-coordinates>
          </map-polygon>
          <map-linestring class="line">
            <map-coordinates>-75.6168365 45.471929 -75.6855011 45.458445 -75.7016373 45.4391764 -75.7030106 45.4259255 -75.7236099 45.4208652 -75.7565689 45.4117074 -75.7833481 45.384225 -75.8197403 45.3714435 -75.8516693 45.377714</map-coordinates>
          </map-linestring>
          <map-point class="point">
            <map-coordinates>-75.6916809 45.4186964</map-coordinates>
          </map-point>
        </map-geometrycollection>
      </map-geometry>
      <map-properties><h2>This is a Geometry Collection</h2></map-properties>
    </map-feature>
  </layer->
    </mapml-viewer>
```